lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
```

Comment:
hey @uny! sorry, i think maybe i wasn't clear enough. i think that **instead** of having a `command` option, we should be explicit about the available commands by creating an option for each one. this fix still allows someone to enter `lksjfsldksdjr` for the `command` parameter, which will result in an error. 

if we create an option for `build` and `bootstrap` we will have covered all of our bases more explicitly.

thanks!